2-functions/2-arguments/2-after.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+final class ReportGenerator
+{
+
+    // ...
+
+    public function generateSalesReport(User $user, DateRange $range)
+    {
+        $reportData = $this->repository->getSalesReportByUser($user->getId(), $range);
```

Comment:
```suggestion
        $reportData = $this->salesReportRepository->findByUser($user->getId(), $range);
```
Would this be too much..?

Reply:
Yes ;)

Reply:
lol